Makefile

```diff
@@ -219,6 +224,15 @@ test-back: ## run back-end tests
 	@DB_PORT=$(DB_PORT) bin/pytest
 .PHONY: test-back
 
+# -- Cookie cutter
+
+start-project: ## bootstrap a new project based on Richie using Cookiecutter
```

Comment:
Also: not a big fan of this strategy with the docker-compose volume trick that forces to create a repository a the same level/path as a cloned Richie project.

Reply:
@jmaupetit not sure what I can do better for this command inside richie... generate the project inside richie would be worse ? Asking for the location is not a possbile in a make command... I could make a bin but it would be less straightforward... Do you have something in mind?

Reply:
Yes. IRL discussion is on-going...